*** ./smali/com/android/internal/policy/impl/LockPatternKeyguardView$3.smali	2013-07-31 22:23:28.387405316 +0000
--- /home/daniel/patchrom/MIUIv5_protou/temp/new_smali/android.policy.jar.out/./smali/com/android/internal/policy/impl/LockPatternKeyguardView$3.smali	2013-07-31 22:21:57.640731076 +0000
***************
*** 7,14 ****
  
  
  # annotations
! .annotation system Ldalvik/annotation/EnclosingClass;
!     value = Lcom/android/internal/policy/impl/LockPatternKeyguardView;
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
--- 7,14 ----
  
  
  # annotations
! .annotation system Ldalvik/annotation/EnclosingMethod;
!     value = Lcom/android/internal/policy/impl/LockPatternKeyguardView;->createKeyguardScreenCallback()Lcom/android/internal/policy/impl/KeyguardScreenCallback;
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
***************
*** 466,472 ****
  
      .local v4, showTimeout:Z
      :goto_3
!     if-eqz v5, :cond_5
  
      iget-object v8, p0, Lcom/android/internal/policy/impl/LockPatternKeyguardView$3;->this$0:Lcom/android/internal/policy/impl/LockPatternKeyguardView;
  
--- 466,472 ----
  
      .local v4, showTimeout:Z
      :goto_3
!     #if-eqz v5, :cond_5
  
      iget-object v8, p0, Lcom/android/internal/policy/impl/LockPatternKeyguardView$3;->this$0:Lcom/android/internal/policy/impl/LockPatternKeyguardView;
  
