*** ./smali/android/preference/PreferenceGroup.smali	2013-08-01 14:28:07.426449928 +0000
--- /home/daniel/patchrom/MIUIv5_protou/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceGroup.smali	2013-08-01 14:26:32.956446865 +0000
***************
*** 257,262 ****
--- 257,264 ----
      :cond_5
      invoke-virtual {p0}, Landroid/preference/PreferenceGroup;->notifyHierarchyChanged()V
  
+     invoke-virtual {p1, p0}, Landroid/preference/Preference;->setParent(Landroid/preference/PreferenceGroup;)V
+ 
      move v1, v2
  
      goto :goto_0
