*** ./smali/android/view/Window.smali	2013-07-31 22:23:13.454070737 +0000
--- /home/daniel/patchrom/MIUIv5_protou/temp/new_smali/framework.jar.out/./smali/android/view/Window.smali	2013-07-31 22:21:49.740730416 +0000
***************
*** 548,553 ****
--- 548,568 ----
  .method public abstract isFloating()Z
  .end method
  
+ .method public isHardwareAccelerated()Z
+     .locals 1
+ 
+     .prologue
+     invoke-virtual {p0}, Landroid/view/Window;->getWindowManager()Landroid/view/WindowManager;
+ 
+     move-result-object v0
+ 
+     invoke-interface {v0}, Landroid/view/WindowManager;->isHardwareAccelerated()Z
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public abstract isShortcutKey(ILandroid/view/KeyEvent;)Z
  .end method
  
