*** ./smali/android/os/Vibrator.smali	2013-08-01 13:46:38.133035905 +0000
--- /home/daniel/patchrom/MIUIv5_protou/temp/new_smali/framework.jar.out/./smali/android/os/Vibrator.smali	2013-08-01 13:45:25.963033565 +0000
***************
*** 26,28 ****
--- 26,39 ----
  
  .method public abstract vibrate([JI)V
  .end method
+ 
+ .method public vibrateEx([B)V
+     .locals 0
+     .parameter "params"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     return-void
+ .end method
