*** ./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-08-01 14:27:44.169782507 +0000
--- /home/daniel/patchrom/MIUIv5_protou/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-08-01 14:26:09.819779449 +0000
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract getAdnCapacity()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract getAdnRecordsInEf(I)Ljava/util/List;
      .annotation system Ldalvik/annotation/Signature;
          value = {
***************
*** 37,42 ****
--- 45,58 ----
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract getFreeAdn()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
